enatega-multivendor-app/translations/zh.js

```
@@ -619,5 +619,6 @@ welcomeText: '欢迎来到Enatega',
   "kindlyBeAdvisedText": "请注意，您的留言可能也会对负责将您的订单送达场地的快递合作伙伴可见。",
   exploreYallaCities: '探索Yalla城市',
   Card: '卡',
+  OrderPriceValidation:"您的订单金额低于餐厅的最低订单要求。请添加更多商品到购物车中。"
```

Biome (1.9.4)

[error] This property value named AddTip is later overwritten by an object member with the same name.

Overwritten with this value.

If an object property with the same name is defined multiple times (except when combining a getter with a setter), only the last definition makes it into the object and previous definitions are ignored.
Unsafe fix: Remove this property value named AddTip


(lint/suspicious/noDuplicateObjectKeys)